addons/website_payment/data/ir_action_data.xml

```diff
@@ -10,4 +10,12 @@ menu_id = env.ref('website.menu_website_website_settings').id
 action = env.company._run_payment_onboarding_step(menu_id=menu_id)
         </field>
     </record>
+    <record id="action_activate_stripe" model="ir.actions.server">
+        <field name="name">Activate Stripe</field>
+        <field name="model_id" ref="website_payment.model_payment_provider"/>
+        <field name="state">code</field>
+        <field name="code">
+action = model.activate_stripe()
+        </field>
```

Comment:
Could be one line. 
```suggestion
        <field name="code">action = model.activate_stripe()</field>
```

Also, I don't think you need to define `action` :thinking:  (but not sure so leave as is as it is done everywhere like this)